src/frontend/styles/components/sidebar.pcss

```diff
@@ -73,11 +73,45 @@
     }
   }
 
+  &__search {
+    @apply --input;
+
+    appearance: none;
+    background: url("../svg/search.svg") left 10px center no-repeat;
+    background-color: var(--color-input-primary);
+    border: 1px solid transparent;
+    border-radius: 8px;
+    padding: 8px 35px;
+    padding-right: 45px;
+    line-height: 21px;
+
+    &-wrapper-mac::after {
+      color: var(--color-button-secondary);
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 17px;
+      content: "⌘ P";
+      margin-left: -50px;
+    }
+
+    &-wrapper-other::after {
+      color: var(--color-button-secondary);
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 17px;
+      content: "Ctrl P";
+      margin-left: -50px;
+    }
```

Comment:
duplicated code. You can move content to the some data attribute, for example `data-shortcut` and then access it in CSS:

```CSS
content: attr(data-shortcut)
```

Reply:
or use separate element (`__search-shortuct`) and change its textContent

Reply:
done